universal-application-tool-0.0.1/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -126,6 +127,23 @@ public void index_usesMessagesForUserPreferredLocale() {
     assertThat(contentAsString(result)).contains("obtener beneficios");
   }
 
+  @Test
+  public void index_missingTranslationForProgram_defaultsToEnglish() {
+    resourceCreator.insertProgram(
+        Locale.forLanguageTag("es-US"), "Programa en español", LifecycleStage.ACTIVE);
+    resourceCreator.insertProgram("English program", LifecycleStage.ACTIVE); // Missing translation
+
+    // Set the PLAY_LANG cookie
+    Http.Request request =
+        fakeRequest().langCookie(Locale.forLanguageTag("es-US"), stubMessagesApi()).build();
```

Comment:
Thought (blocking): it might be more obvious to use a non-US locale here for the test.

Reply:
Good point! I changed it to `Locale.FRANCE`

Reply:
Never mind, I forgot the locale needs to be one of our supported locales, which so far only include `es-US` and `en-US`